Apps/Playground/Scripts/experience.js

```diff
@@ -156,6 +157,45 @@ CreateBoxAsync().then(function () {
                     }, 5000);
                 }
 
+                if (xrFeaturePoints) {
+                    const xrFeaturePointsModule = xr.baseExperience.featuresManager.enableFeature(
+                        BABYLON.WebXRFeatureName.FEATURE_POINTS,
+                        "latest",
+                        {});
+
+                    var pcs= new BABYLON.PointsCloudSystem("pcs", 5, scene);
+                    var featurePointInitFunc = function (particle, i, s) {
```

Comment:
I probably don't have the right context but what's experience.js file supposed to represent? Is this examples of how client is expected to consume this?

I'm going to assume that's the case - I'm confused about is why one would need to create and initiate the feature points. If this is bootstrapping code just for the purpose of illustrating usage in some sandbox context, then i think comments would be super useful.

Reply:
Experience.js is a sample app we use for testing and to show off how to use some of the features.  The reason we need to initiate the feature here is because this is essentially just a babylonjs script like any consuming application would use. 

I'll add some comments around what is going on here for the point cloud system.

Reply:
No urgency to this, but at some point in the future the Playground app is likely to evolve into something a little different from what it's being used for right now. That's a ways into the future, but it might be good to get a head start on thinking about how we can keep our test harnesses for new features without necessarily checking them all into this one file. I'll start giving that a little thought; please let me know if anybody has any preferences or cool ideas to try.